src/components/DataTable/table.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../styles/index.scss';
+
+// STYLES FOR Table.js
+
+.table .thead-light th, .table .thead-dark th {
```

Comment:
Try to avoid use of the themes in the CSS.
If avoiding is not possible, then use root selector.

Reply:
@petrKavulok ^^^